docs/buildah-manifest-add.md

```diff
@@ -36,6 +36,22 @@ the image.  If *imageName* refers to a manifest list or image index, the
 architecture information will be retrieved from it.  Otherwise, it will be
 retrieved from the image's configuration information.
 
+**--authfile** *path*
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
+
+**--cert-dir** *path*
+
+Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry.
+Default certificates directory is _/etc/containers/certs.d_.
```

Comment:
```suggestion
The default certificates directory is _/etc/containers/certs.d_.
```

Reply:
I'm fine adding this suggestion, and the next one. Keep in mind I copied that from the `manifest push` man page.

If you don't mind I would ensure the style is consistent across all the man pages of buildah.